e2e/tests/test_cases/uninstallation_checks.robot

```diff
@@ -0,0 +1,75 @@
+*** Settings ***
+Documentation    Uninstallation Checks
+
+Test Tags    manual_test_case
+
+Resource    ../keywords/common.resource
+Resource    ../keywords/setting.resource
+Resource    ../keywords/volume.resource
+Resource    ../keywords/persistentvolume.resource
+Resource    ../keywords/persistentvolumeclaim.resource
+Resource    ../keywords/workload.resource
+Resource    ../keywords/backup.resource
+Resource    ../keywords/snapshot.resource
+Resource    ../keywords/backupstore.resource
+Resource    ../keywords/longhorn.resource
+Library     ../libs/keywords/setting_keywords.py
+
+Test Setup    Set test environment
+Test Teardown    Cleanup test resources
+
+*** Variables ***
+${LOOP_COUNT}    1
+${RETRY_COUNT}    300
+${RETRY_INTERVAL}    1
+${DATA_ENGINE}    v1
+${LONGHORN_BRANCH}    master
```

Comment:
It would be ideal if the installation parameters, such as `LONGHORN_REPO_BRANCH`, `CUSTOM_LONGHORN_MANAGER_IMAGE`, `CUSTOM_LONGHORN_ENGINE_IMAGE`, could be retrieved from Jenkins environment variables, and passed to a shell script for installing Longhorn using different method, similar to what our `test_upgrade` pytest and `upgrade_longhorn.sh` do. Do you think it's possible for you to implement this?

Reply:
Updated, added `e2e/utilities/longhorn-setup.sh` to get  environment variable to customize test images. I tested on local and Jenkins

[report.zip](https://github.com/user-attachments/files/17009715/report.zip)

![image](https://github.com/user-attachments/assets/e4d9be9f-7c78-4bba-9946-e0bf454c882c)

Reply:
Why is adding `e2e/utilities/longhorn-setup.sh` necessary to obtain the environment variables for custom images? Aren't those variables already passed to the container in `pipeline/e2e/Jenkinsfile` at https://github.com/longhorn/longhorn-tests/blob/master/pipelines/e2e/Jenkinsfile#L95?